engine/src/test/java/com/redhat/cloud/notifications/recipients/itservice/ITUserServiceTest.java

```diff
@@ -57,9 +57,12 @@ void shouldPickPrimaryEMailAsUsersEmail() {
         List<ITUserResponse> itUserResponses = List.of(itUserResponse);
 
         RbacRecipientUsersProvider rbacRecipientUsersProvider = new RbacRecipientUsersProvider(itUserServiceWrapper, new SimpleMeterRegistry());
```

Comment:
`RbacRecipientUsersProvider` is instantiated with `new` instead of being injected as a CDI bean. As a consequence, the `@PostConstruct` method is never called and the retry policy is never initialized, which leads to a test failure.